forecast_solar/__init__.py

```diff
@@ -34,6 +34,7 @@ class ForecastSolar:
     damping: float = 0
     session: ClientSession | None = None
     ratelimit: Ratelimit | None = None
+    inverter: float | None = None
```

Comment:
What value are you entering here? Watts or kVA? Considering 1 kVA = 1000 Watts.

Reply:
watts is what the api wants

Reply:
Maybe use `int` as typing 😄

Reply:
Actually, just tested it again manually by calling the url (After my ratelimit reset :laughing:) and it wants kW :facepalm:

Reply:
I may have an `api_key` for you so you can test more often 😉